frontend/components/SetupOnRamp/OnRampPaymentSteps/useCreateAndTransferFundsToMasterSafeSteps.tsx

```diff
@@ -169,6 +184,14 @@ export const useCreateAndTransferFundsToMasterSafeSteps = (
     };
   }
 
+  // If safe already exists, mark as completed and return empty steps (deposit flow)
```

Comment:
It doesn't need to know which flow it is so..
```suggestion
  // If safe already exists, mark as completed and return empty steps
```